lib/ocpp/include/ocpp_i.h

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
You have only one file in include directory which looks quite pointless, also the src directory has couple of files. I suggest putting all the files to `ocpp/` directory.